src/ProofOfConcepts/ACLFilterChips/customStyles.css

```diff
@@ -49,32 +34,31 @@
   color: rgb(234, 0, 255);
 }
 
-.pf-c-chip-group.bkg-color {
+:root .pf-c-chip-group.mas-m-resource-group {
   background-color: var(--pf-c-label--BackgroundColor);
-  --pf-c-label--BackgroundColor: var(--pf-global--palette--blue-100);
-}
 
-.pf-c-chip-group__close {
-  display: flex;
-  align-self: center;
+  --pf-c-chip-group--m-category--PaddingTop: 0;
+  --pf-c-chip-group--m-category--PaddingBottom: 0;
 }
 
-.pf-c-chip.custom-c-chip {
-  --pf-c-chip__c-button--PaddingTop: 0;
-  --pf-c-chip__c-button--PaddingRight: 0;
-  --pf-c-chip__c-button--PaddingBottom: 0;
-  --pf-c-chip__c-button--PaddingLeft: 0;
+.pf-c-chip.mas-m-custom-chip {
   --pf-c-chip--PaddingTop: var(--pf-global--BorderWidth--sm);
   --pf-c-chip--PaddingRight: var(--pf-global--BorderWidth--sm);
   --pf-c-chip--PaddingBottom: var(--pf-global--BorderWidth--sm);
   --pf-c-chip--PaddingLeft: var(--pf-global--BorderWidth--sm);
-}
 
-.pf-c-chip.custom-c-chip {
   --pf-c-chip__text--MaxWidth: auto;
+  display: table-row;
+  background-color: var(--pf-c-label--BackgroundColor);
+  --pf-c-label--BackgroundColor: var(--pf-global--palette--black-200);
+}
+
+:root .pf-c-chip-group__close {
+  display: flex;
+  align-self: center;
 }
 
-.pf-c-chip-group.bar {
+.pf-c-chip-group.mas-m-bar {
   --pf-c-chip-group--m-category--BackgroundColor: darkgray;
 }
 
```

Comment:
I'm not able to find anything that uses this class. I think it was part of the original POC where we were exploring different possible layout options. Can you check if this is actually needed, and remove if not?